docs/modules/components/pages/processors/ollama_chat.adoc

```diff
@@ -0,0 +1,123 @@
+= ollama_chat
+:type: processor
+:status: experimental
+:categories: ["AI"]
+
+
+
+////
+     THIS FILE IS AUTOGENERATED!
+
+     To make changes, edit the corresponding source file under:
+
+     https://github.com/redpanda-data/connect/tree/main/internal/impl/<provider>.
+
+     And:
+
+     https://github.com/redpanda-data/connect/tree/main/cmd/tools/docs_gen/templates/plugin.adoc.tmpl
+////
+
+
+component_type_dropdown::[]
+
+
+Processor that uses the Ollama API to generate text.
+
+Introduced in version 4.31.0.
+
+
+[tabs]
+======
+Common::
++
+--
+
+```yml
+# Common config fields, showing default values
+label: ""
+ollama_chat:
+  server_address: "" # No default (optional)
+  model: llama3 # No default (required)
+  prompt: "" # No default (optional)
+```
+
+--
+Advanced::
++
+--
+
+```yml
+# All config fields, showing default values
+label: ""
+ollama_chat:
+  server_address: "" # No default (optional)
+  ollama_directory: "" # No default (optional)
+  model: llama3 # No default (required)
+  prompt: "" # No default (optional)
+  system_prompt: "" # No default (optional)
+```
+
+--
+======
+
+Processor that uses the Ollama API to generate text.
+
+See https://ollama.com/[https://ollama.com/^] for more information.
+
+== Fields
+
+=== `server_address`
+
+The address of a remote ollama server to use. If omitted a local ollama server will be started and ran.
+
+
+*Type*: `string`
+
+
+=== `ollama_directory`
+
+The directory to unpack the ollama binary into. Defaults to /tmp
+
+
+*Type*: `string`
+
+
+=== `model`
+
+The ollama model to use
+
+
+*Type*: `string`
+
+
+```yml
+# Examples
+
+model: llama3
+
+model: gemma2
+
+model: qwen2
+
+model: phi3
+```
+
+=== `prompt`
+
+The prompt to submit to the model. By default the entire payload as a string is submitted.
```

Comment:
```suggestion
The prompt you want to generate a response for. By default, the processor submits the entire payload as a string. This field supports [interpolation functions](http://localhost:8000/configuration/interpolation/#bloblang-queries).
```
Just a suggestion to make this description distinct from the system prompt?

Reply:
Thanks, much better!